docs/components/codeblock.md

```diff
@@ -96,6 +96,30 @@ Boolean to control whether to show or hide the search bar and related action but
 />
 ```
 
+### highlightedLineNumbers
+
+An array of line numbers to initially highlight in the code block. 
+
+If search is enabled, matching results will be highlighted in place of the provided line numbers, and all highlighted lines will be cleared once the search query is cleared.
+
+<ClientOnly>
+  <KCodeBlock
+    id="code-block-highlight"
+    :code="code"
+    language="json"
+    :highlighted-line-numbers="[2,4,6]"
+  />
+</ClientOnly>
+
+```html
+<KCodeBlock
+  id="code-block-highlight"
+  :code="code"
+  language="json"
+  :highlighted-line-numbers="[2,4,6]"
```

Comment:
Shall we also support the syntax that [VitePress](https://vitepress.dev/guide/markdown#line-highlighting-in-code-blocks) and [Gatsby](https://www.gatsbyjs.com/plugins/gatsby-remark-highlight-code/#lines-highlighting) supports (`4,7-13,16,23-27,40`) so that it's easier to highlight a range of lines?

Reply:
Tracking in https://konghq.atlassian.net/browse/KHCP-14552